routers/routes/routes.go

```diff
@@ -923,7 +923,7 @@ func RegisterRoutes(m *macaron.Macaron) {
 					m.Post("/", lfs.PostLockHandler)
 					m.Post("/verify", lfs.VerifyLockHandler)
 					m.Post("/:lid/unlock", lfs.UnLockHandler)
-				}, context.RepoAssignment())
```

Comment:
Why we remove this?

Reply:
LFS has a different authentication scheme to that of the normal repos. RepoAssignment simply doesn't work for it.

Reply:
Yes, we could/should make LFS's authentication a macaron handler but we should do that as part of a larger refactor of the LFS server - along with a check that we are not missing functionality from git-lfs's lfstest-gitserver.

But that's another pr.